tests/functional/test_install.py

```diff
@@ -53,19 +51,33 @@ def test_pep518_with_user_pip(script, virtualenv, pip_src, data):
     system_pip_dir.mkdir()
     with open(system_pip_dir / '__init__.py', 'w') as fp:
         fp.write('raise ImportError\n')
-    to_install = data.src.join("pep518-3.0")
-    for command in ('install', 'wheel'):
-        kwargs = {}
-        if sys.version_info[:2] == (3, 3):
-            # Ignore Python 3.3 deprecation warning...
-            kwargs['expect_stderr'] = True
-        script.run(
-            "python", "-c",
-            "import pip._internal; pip._internal.main(["
-            "%r, " "'-f', %r, " "%r, "
-            "])" % (command, str(data.packages), str(to_install)),
-            **kwargs
-        )
+    kwargs = {}
+    if sys.version_info[:2] == (3, 3):
+        # Ignore Python 3.3 deprecation warning...
+        kwargs['expect_stderr'] = True
+    script.run(
+        "python", "-c",
+        "import pip._internal; pip._internal.main(['wheel', '-f', %r, %r])"
```

Comment:
You've removed a test-run for "install" command here.

Reply:
Yes, as noted in the commit message the goal of this test is to check the command to install build dependencies can be run successfully, there's really no need to check with both install and wheel commands.

Reply:
Fair enough. :)